pkg/testrunner/run.go

```diff
@@ -137,3 +139,23 @@ func GetArgoURL(tmClient kubernetes.Interface, tr *tmv1beta1.Testrun) (string, e
 
 	return argoUrl.String(), nil
 }
+
+// GetClusterDomainURL tries to derive the cluster domain url from an grafana ingress if possible. Returns an error if the ingress cannot be found or is in unexpected form.
+func GeClusterDomainURL(tmClient kubernetes.Interface) (string, error) {
+	// try to derive the cluster domain url from grafana ingress if possible
+	// return err if the ingress cannot be found
+	ingress, err := tmClient.Kubernetes().ExtensionsV1beta1().Ingresses("monitoring").Get("grafana", v1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("cannot get grafana ingress: %v", err)
+	}
+	if len(ingress.Spec.Rules) == 0 {
+		return "", fmt.Errorf("cannot get ingress rule from ingress %v", ingress)
+	}
+	host := ingress.Spec.Rules[0].Host
+	r, _ := regexp.Compile("[a-z]+\\.ingress\\.(.+)$")
```

Comment:
please handle the error

Reply:
I intentionally did not: the regex is static -> cannot change during runtime -> cannot error during runtime

Reply:
oh yes you're right 👍